### PR TITLE
Updating key pairs in data parser

### PR DESCRIPTION
**Pull request by anwarMZ**
Few updates to reflect changes in gvf files for visualization:

- Updated attribute `status` to distinguish between _actively_circulating_ and _de-escalated_ variants. `status` was previously used to distinguish between VOCs, VOIs and VUM.
- changed the attribute `who_variant` to `variant`  because WHO is not updating variant list as often and we need to keep up with different sources so removing WHO tag
- `variant_type` is now replacing `status` to distinguish VOCs, VOIs ad VUMs

### CODE REVIEW

**Review by ivansg44 — CHANGES_REQUESTED** [review]
I think you should add the new gvf files and surveillance reports to `reference_data_/` and `reference_surveillance_reports/`, and I should implement a visual encoding for de-escalating and circulating strains, before we merge this PR.

File: data_parser.py [hunk 1]
```diff
@@ -295,6 +304,10 @@ def get_data(dirs, show_clade_defining=False, hidden_strains=None,
             voc_strains,
         "voi_strains":
             voi_strains,
+        "de_escalated_strains":
+            de_escalated_strains,
+        "circulating_strains":
+            circulating_strains,
```
[the remark this answers is not in the document]
Comment (anwarMZ) [reply data_parser.py:310]: Yes, we will differentiate these in the visualization too.

[the remark this answers is not in the document]
Comment (ivansg44) [reply data_parser.py]: Resolved with 099e341

**Review by ivansg44 — APPROVED** [review]
I'm going to merge this

**Pull request merged**